jobs/handler.go

```diff
@@ -9,6 +9,7 @@ import (
 type appealService interface {
 	Find(*domain.ListAppealsFilter) ([]*domain.Appeal, error)
 	Revoke(id, actor, reason string) (*domain.Appeal, error)
+	Create(appeals []*domain.Appeal) error
```

Comment:
Is this required?

Reply:
I made this change, as I was thinking of mocking appealService itself for a usecase. Didn't need it later, but left it as it is, since it felt that this interface should have the contract of create appeal also.

Reply:
we're trying to have interface defined in the user part and only define methods that the user's using as per [this docs](https://github.com/golang/go/wiki/CodeReviewComments#interfaces)